frontend/src/app/AppRouting.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import {Route } from "react-router-dom";
+import Home from './Pages/Home'
+import About from './Pages/About'
+
+function AppRouting(){
```

Comment:
There will be a lot of routes and this file will get bloated. We should use a json object with routing configurations and map all of these to Routes as it is done [here](https://alligator.io/react/react-router-map-to-routes/). Also please see [this](https://itnext.io/async-react-using-react-router-suspense-a86ade1176dc). We will fetch data at the entry of each route so the components should be async.